packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3237,6 +3240,7 @@ export interface VideoGalleryProps {
     onRenderLocalVideoTile?: (localParticipant: VideoGalleryLocalParticipant) => JSX.Element;
     onRenderRemoteVideoTile?: (remoteParticipant: VideoGalleryRemoteParticipant) => JSX.Element;
     onUnpinParticipant?: (userId: string) => void;
+    overflowGalleryLayout?: OverflowGalleryLayout;
```

Comment:
nit this is already a VideoGallery so `Gallery` isn't needed in the naming here, to match the existing `layout` props should this instead be `overflowLayout: OverflowGalleryLayout`?

Reply:
Really good point here. This new prop will pertain to how VideoGallery will layout overflow participants and overflowLayout would be a concise name. But we might want to keep the naming connection of inner overflow components HorizontalGallery, VerticalGallery etc to this new prop so overflowGalleryLayout maintains a stronger connections. We can always update it in beta if we start getting signals we are exporting the components if ARB picks on it.